CHANGELOG.md

```diff
@@ -6,6 +6,7 @@ This document outlines major changes between releases.
 
 ### Added
 - Exposure of pool metrics (#615)
+- Configuration of `set_copies_number` -- number of the object copies to store (#634)
```

Comment:
> number of the object copies to store

Actually it isn't true. It's a copies number required to consider put successful and return ok. In the background neofs-node still makes replication according to container policy.

I think we can drop  `-- number of the object copies to store`